readme.md

```diff
@@ -14,6 +14,12 @@ This package:
s="string"></td><td class="string"></td><td class="number">492</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.15.0">v0.15.0</a></td><td class="string">2014-07-29</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">441</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.14.3">v0.14.3</a></td><td class="string">2014-07-27</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">346</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.14.2">v0.14.2</a></td><td class="string">2014-07-25</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">334</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.14.1">v0.14.1</a></td><td class="string">2014-07-24</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">325</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.14.0">v0.14.0</a></td><td class="string">2014-07-22</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">403</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.13.3">v0.13.3</a></td><td class="string">2014-06-25</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">2209</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.13.2">v0.13.2</a></td><td class="string">2014-06-18</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">353</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.13.1">v0.13.1</a></td><td class="string">2014-06-14</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">363</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.13.0">v0.13.0</a></td><td class="string">2014-06-05</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">713</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.12.7">v0.12.7</a></td><td class="string">2014-05-27</td><td class="undefined"></td><td class="boolean">false</td><td class="string"></td><td class="string"></td><td class="number">397</td></tr><tr><td class="string"><a href="https://github.com/electron/electron/releases/tag/v0.12.6">v0.12.6</a></td><td class="string">2014-05-26</td><td cl
```

Comment:
Does <table><tr><td> work inside of markdown files? Is there a way to have the generated content use markdown's table notation instead of html?

Reply:
@ckerr it looks like your comment formatting got messed up there ☝️ 

Yes we could absolutely do markdown tables instead. This was just an easier route.

Reply:
This would do the trick: https://github.com/tsuz/json-to-markdown-table

Reply:
Good call on the markdown, @ckerr. Less noisy.